kava-protocol-home.js

```diff
@@ -80,7 +80,7 @@ const displayInThousands = (value) => {
 }
 
 const isKavaNativeAsset = (denom) => {
-  return ['ukava-a', 'usdx', 'hard', 'ukava', 'hard-a', 'swp-a'].includes(denom)
+  return ['ukava-a', 'usdx', 'hard', 'ukava','kava-a', 'hard-a', 'swp-a'].includes(denom)
```

Comment:
When would we encounter a cdp type of `kava-a`?

Reply:
the new endpoints is returning kava for the denom name I believe I can check the exact one for you.